queue_processors/autoreduction_processor/reduction_exceptions.py

```diff
@@ -0,0 +1,28 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Exceptions that may be raised when performing a reduction
+"""
+
+
+class DatafileError(Exception):
+    """
+    Exception when there is an issue reading a datafile
+    """
+
+
+class ReductionScriptError(Exception):
+    """
+    Exception raised when an unhandled exception is produced within the reduction script
+    """
+
+
+class SkippedRunException(Exception):
+    """
+    Exception for runs that have been skipped
+    Note: this is currently only the case for EnginX Event mode runs at ISIS
```

Comment:
```suggestion
    Exception for runs that have been skipped
```

Reply:
when this code first was written it was probably only an exception for EnginX. However now we label everything that is not an exceptable RB number as skipped